config/helm/chart/default/questions.yml

```diff
@@ -3,29 +3,164 @@ categories:
   - Monitoring
 questions:
 
-  - variable: crdApplied
-    label: "CRD manually applied"
-    description: "Please make sure you have applied the latest CRD by following the instructions in the description above."
-    default: ""
-    required: true
-    type: enum
-    options:
-      - "yes"
-    group: "Prerequisites"
-
   #################### Global Configuration ####################
+  - variable: installCRD
+    label: "Install Custom Resource Definitions"
+    description: "Installs the Custom Resource Definitions for the Dynakube. This is recommended if you haven't installed it manually yet. Default: true"
+    default: true
+    type: boolean
+    group: "Global Configuration"
+
   - variable: operator.image
-    label: "Set a custom image for operator"
-    description: "Optional: Set a custom image for operator. Defaults to docker.io/dynatrace/dynatrace-operator"
+    label: "Set a custom image for operator components"
+    description: "Set a custom image for operator. Defaults to docker.io/dynatrace/dynatrace-operator"
```

Comment:
why remove the optional?

Reply:
everything is optional, felt redundant 
should I add `Optional` to every field ?

Reply:
you're right, not needed

Reply:
I could update the `app-readme.md` so it say that the configuration fields are optional
or have the section names have the word `Optional`